generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
There is no default value for 'expected_hugepage_size' in debug.log if we don't set it.   And setup_hugepages = yes as configured in cfg. 

The case actually ran with 2M hugepage in my test. 
cmdline:
-m 4096 -mem-path /mnt/kvm_hugepage 
# mount
none on /mnt/kvm_hugepage type hugetlbfs (rw,relatime,seclabel,pagesize=2048K)